admin/app/controllers/authenticated/certifications/certification/details.js

```diff
@@ -0,0 +1,18 @@
+import Controller from '@ember/controller';
+
+export default class DetailsController extends Controller {
+  assignQuestionNumberForDisplay(model) {
+    let i = 1;
```

Comment:
Commentaire fait par Marianne 

```suggestion
    let index = 1;
```